frame/bags-list/src/mock.rs

```diff
@@ -112,6 +119,16 @@ impl ExtBuilder {
 		self
 	}
 
+	/// Set vote weights of some Id's used for special case
+	pub(crate) fn add_aux_data(mut self) -> Self {
```

Comment:
Honestly, I am against this. This is data specific to one test, and should not pollute the entirety of the mock file just for that. Note that this is really coming from experience -- maintaining a clean, readable and sensible mock file is also important. Just look at staking if you think otherwise :D

Reply:
927a3836e28c4f05de8eb29a1caed13cec285ee3